plugins/package-managers/stack/src/main/kotlin/Stack.kt

```diff
@@ -347,7 +347,7 @@ class Stack(
                 .map(String::trim)
                 .filter(String::isNotEmpty)
                 .mapNotNullTo(mutableSetOf(), ::parseAuthorString),
-            declaredLicenses = map["license"]?.let { setOf(it) } ?: emptySet(),
+            declaredLicenses = map["license"]?.let { setOf(it) }.orEmpty(),
```

Comment:
How about `.orEmpty().toSet()` to get rid of this `let` ?

Reply:
See above.